src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/casenotes/CaseNotesService.kt

```diff
@@ -108,22 +109,29 @@ class CaseNotesService(
   /**
    * For reconciliation or migration
    */
-  fun getCaseNotes(bookingId: Long): BookingCaseNotesResponse {
-    offenderBookingRepository.findById(bookingId).orElseThrow { NotFoundException("bookingId $bookingId not found") }
-    return BookingCaseNotesResponse(
-      offenderCaseNoteRepository.findAllByOffenderBooking_BookingId(bookingId).map { it.toCaseNoteResponse() },
+  fun getCaseNotes(offenderNo: String): PrisonerCaseNotesResponse {
+    offenderRepository.findByNomsId(offenderNo).takeIf {
```

Comment:
think you can just do an `ifEmpty` here instead?

Reply:
ah cool, ta. Will change next time